content-src/components/StartupOverlay/StartupOverlay.jsx

```diff
@@ -12,12 +12,26 @@ export class _StartupOverlay extends React.PureComponent {
     this.initScene = this.initScene.bind(this);
     this.removeOverlay = this.removeOverlay.bind(this);
     this.onInputInvalid = this.onInputInvalid.bind(this);
+    this.setFormFields = this.setFormFields.bind(this);
 
-    this.state = {emailInput: ""};
+    this.state = {emailInput: "", flowId: "", flowBeginTime: ""};
     this.initScene();
   }
 
   initScene() {
+    let req = new XMLHttpRequest();
+    req.onreadystatechange = () => {
+      if (req.readyState === 4 && req.status === 200) {
+        this.setFormFields(req.responseText);
+      }
+    };
+    req.onerror = e => {
+      console.error("fxa error", e);
```

Comment:
Should we report these to telemetry?

Reply:
sure!

Reply:
@vladikoff, would fxa use/want this data?

Reply:
@ericawright not for now, worst case we can look at telemetry I guess!

Reply:
@vladikoff that's what I mean, if I store this event in telemety would you use it, since AS doesn't need it.

Reply:
@ericawright oh, if it is easy enough and you have other errors already sent to telemetry let's do it! We can later look if the request fails for users